pages/partnerships/partnerships.html

```diff
@@ -0,0 +1,422 @@
+             <md-card-title>
+              <md-card-title-text>
+                <h1 class="md-display-3" hide-xs>
+                  We're looking for partners
+                </h1>
+                <h3 class="md-display-1" show-xs hide-gt-xs>
+                  We're looking for partners
+                </h3>
+              </md-card-title-text>
+              </md-card-title>
+              <h4 class="md-headline partnerships-page-title-box" hide-xs>
+                The Oppia Foundation has already helped over 800,000 learners. Help us bring engaging, effective lessons to students around the worlds.
+              </h4>
+              <h6 strong class="md-headline partnerships-page-title-box" show-xs hide-gt-xs>
+                The Oppia Foundation has already helped over 800,000 learners. Help us bring engaging, effective lessons to students around the worlds.
+              </h6>
+          </md-content>
+        </md-card-content>
+      </div>
+      <div flex-gt-md="50" flex="100" layout="column" layout-align="center" layout-wrap>
+        <div hide-xs hide-sm hide-md class="bottom-box partnerships-page-welcome-image-container">
+          <img layout="column" layout-align="center-end" src="/static/images/partnerships_donate_hand1.png" class="partnerships-page-donate-hand-image">
+        </div>
+      </div>
+    </div>
+  </md-card>
+  <md-card layout-fill class="oppia-foundation-gray-background oppia-foundation-vertical-padding partnerships-page-conditional-horizontal-padding">
+    <div layout-gt-md="row" layout-xs="column" layout-align="space-between none" flex layout-fill>
+      <div hide-gt-md flex-gt-md="50" flex="100" layout="column" layout-align="center" layout-wrap>
+        <img src="https://dummyimage.com/512x512/000/fff" class="partnerships-page-place-holder-image">
+      </div>
+      <div flex-gt-md="50" flex="100">
+        <md-card-content layout="column">
+          <md-content layout="column" layout-align="center none" class="oppia-foundation-gray-background oppia-foundation-black-text">
+             <h3 class="md-display-1 partnerships-description-headline-padding">
+                Empowering students in every community
+            </h3>
+            <p>
+              Overcrowded classrooms, scarce materials, outdated teaching methods and excessive punishments prevent students in underserved communities from acquiring the knowledge they need to be successful later in life.
+            </p>
+            <p>
+              It’s a problem that isn’t limited to developing countries. In the united states, low income and rural students consistently show major achievement gaps compared to their more affluent peers.
+            </p>
+            <p>
+              The Oppia Foundation believes a digital classroom that provides access to interactive, engaging lessons is the key to closing these gaps in educational outcomes.
+            </p>
+          </md-content>
+      </md-card-content>
+      </div>
```

Comment:
Something's wrong with the indentation (here and below)

Reply:
Fixed! I have checked each line below this line.